README.md

```diff
@@ -1,2 +1,422 @@
-# flutter-example-automation
-Basic Flutter app with askui automation examples
+# Askui automation example with Flutter
+This repository contains examples of using [askui](https://www.askui.com/) to automate an android app built with [Flutter](https://flutter.dev/). To run the example, setup the provided demo app by following the instruction. This example assumes that you already have the [Android Emulator](https://developer.android.com/studio/run/emulator) installed.
```

Comment:
Sorry for the nitpicking here:
 
 setup -> set up
 instruction -> instructions